FAQ.md

```diff
@@ -79,3 +79,11 @@ from the API groups that it needs (be they Kubernetes types or your own).
 See the [scheme builder
 docs](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/scheme) for
 more information.
+
+### Q: How to use third party APIs in `envtest`?
```

Comment:
This could match the actual error better
```suggestion
### Q: When using envtest, why do I see the error: `no matches for kind "Kind" in version "<group>/<version>"`?
```

Reply:
That's the error for the CRD not being found, there's a different error for the scheme not being registered I think?

Reply:
If its not added to scheme, it would be `no "<Kind>" is registered in scheme`, I think. That's explained in the previous question

Reply:
In that case, remove point (1) from my suggestion below.